solutions/fully-configurable/catalogValidationValues.json.template

```diff
@@ -0,0 +1,8 @@
+{
+  "ibmcloud_api_key": $VALIDATION_APIKEY,
+  "existing_resource_group_name": "geretain-test-secrets-manager",
+  "service_plan": "trial",
+  "kms_encryption_enabled": true,
+  "existing_kms_instance_crn": $HPCS_US_SOUTH_CRN,
+  "region": "ca-tor"
```

Comment:
we are trying to keep all KMS enabled instances in `eu-de` (see internal [post](https://ibm-cloudplatform.slack.com/archives/CG06FQ31P/p1742830209625049?thread_ts=1742355323.015669&cid=CG06FQ31P))

Reply:
can you add `"prefix": $PREFIX`